### PR TITLE
#19 fix

### PR DESCRIPTION
**Pull request by trapacska**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -303,23 +305,28 @@ func main() {
...
 	jwtConfig := new(jwt.Config)
 
 	if configs.JSONKeyPath != "" {
-		jsonKeyPth := ""
+		jsonKeyPth := configs.JSONKeyPath
 
-		if strings.HasPrefix(configs.JSONKeyPath, "file://") {
+		url, err := url.Parse(configs.JSONKeyPath)
+		if err != nil {
+			failf("Failed to parse json key path, error: %s", err)
```
Comment (viktorbenei) [inline main.go]: Please don't do this like that. Handle if the path is just a relative path, for any case, not just for p12 or something like that!

Comment (viktorbenei) [inline main.go]: See https://github.com/bitrise-steplib/steps-google-play-deploy/issues/19 for more details

Comment (viktorbenei) [inline main.go]: maybe in this case just do a "file does exist?" check for the path, and if that exists treat it like that

File: main.go [hunk 2]
```diff
@@ -290,6 +292,15 @@ func failf(format string, v ...interface{}) {
 	os.Exit(1)
 }
 
+func getKeyPath(keyPath string) (string, bool, error) {
```
Comment (godrei) [inline main.go]: `func getKeyPath(keyPath string) (string, bool, error) {`

->

`func parse(URL string) (string, bool, error) {`

File: main.go [hunk 3]
```diff
@@ -303,24 +314,26 @@ func main() {
...
 
 	if configs.JSONKeyPath != "" {
-		jsonKeyPth := ""
 
-		if strings.HasPrefix(configs.JSONKeyPath, "file://") {
-			jsonKeyPth = strings.TrimPrefix(configs.JSONKeyPath, "file://")
-		} else {
+		jsonKeyPth, isRemote, err := getKeyPath(configs.JSONKeyPath)
+		if err != nil {
+			failf("Failed to get key path (%s), error: %s", configs.JSONKeyPath, err)
```
Comment (godrei) [inline main.go]: Failed to get key ... -> Failed to parse key ...

File: main.go [hunk 4]
```diff
@@ -331,19 +344,20 @@ func main() {
 		}
 		jwtConfig = authConfig
 	} else {
-		p12KeyPath := ""
+		p12KeyPath, isRemote, err := getKeyPath(configs.P12KeyPath)
+		if err != nil {
+			failf("Failed to get key path (%s), error: %s", configs.P12KeyPath, err)
```
Comment (godrei) [inline main.go]: get -> parse

**Pull request merged**